.gitlab-ci.yml

```diff
@@ -1,14 +1,131 @@
-test:
-  image: "python:3.6-stretch"
-  script:
-    - apt-get update -qq
-    - apt-get install -y git
-    - apt-get install -y xsltproc
-    - apt-get install -y genisoimage
-    - apt-get install -y enchant
-    - apt-get install -y shellcheck
-    - apt-get install -y python3-pip
-    - pip3 install --upgrade pip
-    - pip install tox
-    - tox -e check
-    - tox -e unit_py3_6 "-n $(nproc)"
+stages:
+  - test
+  - doc
+  - package
+  - deploy
```

Comment:
For the first iteration can we please leave out the deployment stage ? The reason is this is done by travis on tags in master and another deployment system will interfere with it. I think if we move the deployment of the doc pages and the package on pip from travis to gitlab-ci it should be done in an extra pull request that deletes the stage in .travis.yml and adds one in .gitlab-ci.yml. Thanks

Reply:
Sure, the deployment stage doesn't really do a lot anyway. It just publishes the current docs from master to gitlab pages, there is no interference with the official docs or the pip package.